README.md

```diff
@@ -129,7 +129,8 @@ All you need is Docker and git (optional).
    ```bash
    # Download and store a local copy of hpe-oneview-sdk-for-puppet and
    # use it as a Docker image.
-   $ docker pull hewlettpackardenterprise/hpe-oneview-sdk-for-puppet:v6.1.0-OV6.1
+   $ docker pull hewlettpackardenterprise/hpe-oneview-sdk-for-puppet:v2.7.0-OV5.3
```

Comment:
I have reverted the docker image to ```v2.7.0-OV5.3```, since that is the last image we have uploaded. And there is no image with version ```v6.1.0-OV6.1```